test/test_fat.rb

```diff
@@ -277,7 +277,10 @@ def test_change_dylib_id
 
       file.write(actual)
 
-      assert equal_sha1_hashes(actual, expected)
+      act = MachO::FatFile.new(actual)
+      exp = MachO::FatFile.new(expected)
+
+      assert_equal exp.dylib_id, act.dylib_id
```

Comment:
Just a reminder: You should be able to revert this and similar changes in the other test methods once `insert_command` is implemented and load command order is preserved again.